Meditation/Meditation/View/Home/SearchPanel.swift

```diff
@@ -0,0 +1,30 @@
+import SwiftUI
+
+struct SearchPanel: View {
+    var body: some View {
+        HStack {
+            VStack(alignment: .leading, spacing: 10) {
+                Text("Good Morning, Oleh")
```

Comment:
Let's do the extra challenge here: localize strings in UI to Ukrainian 😉 

Hint: check how the localization is done in CW and/or ask Vlad.